--- a/src/HtmlBuilder.php
+++ b/src/HtmlBuilder.php
@@ -35,7 +35,7 @@
      * @param \Illuminate\Contracts\Routing\UrlGenerator $url
      * @param \Illuminate\Contracts\View\Factory         $view
      */
-    public function __construct(UrlGenerator $url = null, Factory $view)
+    public function __construct(?UrlGenerator $url = null, Factory $view)
     {
         $this->url = $url;
         $this->view = $view;
--- a/src/FormBuilder.php
+++ b/src/FormBuilder.php
@@ -116,7 +116,7 @@
      * @param  string                                     $csrfToken
      * @param  Request                                    $request
      */
-    public function __construct(HtmlBuilder $html, UrlGenerator $url, Factory $view, $csrfToken, Request $request = null)
+    public function __construct(HtmlBuilder $html, UrlGenerator $url, Factory $view, $csrfToken, ?Request $request = null)
     {
         $this->url = $url;
         $this->html = $html;
